lib/pi_piper/sysfs/version.rb

```diff
@@ -0,0 +1,6 @@
+module PiPiper
+  class Driver; end
```

Comment:
That is exactly my intent, at first...
Our drivers are subclasses of PiPiper::Driver, but it is not defined in gems pi_piper-sysfs (and others...).
Things are difficult when using rake or bundler, when pi_piper is not required before loading the gem code...
it is the only workaround I found... any thoughts ?